flaskServer/restSample.py

```diff
@@ -12,18 +11,16 @@
 class uploadPhoto(Resource):
```

Comment:
To adhere to PEP8 stuff and make this sound more like a class, I may suggest a different name, like "PhotoUpload."